### ISSUE TITLE
Max parallel should be configurable via command line flag

### ISSUE THREAD

**Issue by uded** [wagoodman/bashful#40]
Taking into consideration the purpose of `max-parallel-commands` it should be possible to override the value via a flag. In my case, depending on whether it's my laptop or CI server I might want to limit or raise the number of parallel executions to match my environment better. Having it in the file is not as flexible as I would expect...

**Comment by wagoodman** [comment]
Agreed. I've been contemplating a generic way to override any config option via cli switches without having to explicitly map a cli arg to a config item. I'm open to suggestions.

**Comment by uded** [comment]
OK, I will take a look tomorrow and maybe come up with a simple, elegant solution. Fingers crossed ;-)
